frontend/components/Step/Step.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from "react";
+import { FormContext } from "../Overlay/CreateProject.js";
+import { Files, Success, SetPrice, VerifyHumanity, Submit } from "../Forms";
+
+function Step() {
+  const { activeStepIndex, formData, setFormData } = useContext(FormContext);
+  let stepContent;
+  switch (activeStepIndex) {
+    case 0:
+      stepContent = <Files formData={formData} setFormData={setFormData} />;
```

Comment:
For `formData` and `setFormData ` , both of those pieces of context data can be read directly from context in each of the step components instead of passing the as props. Ex...
```
// Files.js

function Files() {
  const { activeStepIndex, setActiveStepIndex, formData, setFormData } = useContext(FormContext);
...
....
......
```